lib/eslint.js

```diff
@@ -801,10 +798,7 @@ module.exports = (function() {
             }
 
             parseResult = parse(
-                stripUnicodeBOM(text).replace(/^#!([^\r\n]+)/, (match, captured) => {
-                    shebang = captured;
-                    return `//${captured}`;
-                }),
+                stripUnicodeBOM(text).replace(/^#!([^\r\n]+)/, (match, captured) => `//${captured}`),
```

Comment:
Can you explain why they would be different, given that the regexp's are the same? What am I missing?